src/main/java/me/petterim1/scoreboards/ScoreboardUpdater.java

```diff
@@ -22,23 +20,23 @@
 
     @Override
     public void run() {
-        try {
```

Comment:
It would be still good idea to have that try catch there because the async mode is a hacky way to reduce load by doing stuff what you really shouldn't do using async and we don't want to get random errors about that

Reply:
Tested for a long time with many players but never got any errors so far. But alright